### PR TITLE
Update Phpstorm code style settings

### PR DESCRIPTION
**Pull request by iMattPro**

### CODE REVIEW

File: development/development/phpstorm.rst [hunk 1]
```diff
@@ -119,31 +119,24 @@ To set up PHPunit within PhpStorm, go to:
 PhpStorm Setting Exports for phpBB
 ==================================
 
-Copy and save these code blocks as XML files, and they can be imported into PhpStorm's settings to automatically set up most of the configuration recommendations mentioned in this documentation for phpBB.
+What follows are phpBB's code style settings for PhpStorm. Copy and save this code block as ``phpbb.xml``. Then in PhpStorm, go to **Settings > Editor > Code Style**. Click the setting cog icon next to **Scheme* and choose **Import Scheme...** to import the code style recommendations mentioned in this documentation for your phpBB project.
```
Comment (DavidIQ) [inline development/development/phpstorm.rst:122]:
```suggestion
What follows are phpBB's code style settings for PhpStorm. Copy and save this code block as ``phpbb.xml``. Then in PhpStorm, go to **Settings > Editor > Code Style**. Click the setting cog icon next to **Scheme** and choose **Import Scheme...** to import the code style recommendations mentioned in this documentation for your phpBB project.
```

Comment (iMattPro) [reply development/development/phpstorm.rst]: Nice catch

**Comment by marc1706** [comment]
Reference on line 79 does no longer resolve (inspection profile link).

**Comment by iMattPro** [comment]
fixed that

**Review by marc1706 — APPROVED** [review]

**Pull request merged**